enigma-swing/testProject/src/main/java/tests/quilt/issue10/TestClass.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Class to reproduce issue #10
+ * https://github.com/QuiltMC/enigma/issues/10
+ */
+
+package tests.quilt.issue10;
+
+import java.util.function.Function;
+
+public class TestClass {
```

Comment:
Where is this test class used? Is there a unit test for it? What is `testProject`? I dont see how this is even built.